eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -46,7 +50,7 @@ else {
 if ($null -eq [AzureEngSemanticVersion]::ParseVersionString($Version))
 {
     LogError "Version [$Version] is invalid. Please use a valid SemVer"
```

Comment:
It would be nice to create an aka.ms link to the guidelines for change logs so that folks have some information they can read on these various error cases.